xbmc/video/VideoDatabase.cpp

```diff
@@ -270,6 +273,7 @@ void CVideoDatabase::CreateAnalytics()
   m_pDS->exec("CREATE INDEX ix_art ON art(media_id, media_type(20), type(20))");
 
   m_pDS->exec("CREATE INDEX ix_rating ON rating(media_id, media_type(20))");
+  m_pDS->exec("CREATE INDEX ix_uniqueid ON uniqueid(uniqueid_id, media_id, media_type(20), value(20))");
```

Comment:
I think you need multiple indicese. You never query the uniqueid table with a condition covering all four columns. You have multiple SQL queries where you filter by `media_id` and `media_type` and a few on `media_id`, `media_type` and `type`.